test/services/DateTimeServiceSpec.scala

```diff
@@ -26,35 +26,41 @@ class DateTimeServiceSpec extends SpecBase {
 
     "return system date and time for UTC" in new Setup {
       val systemDateTime = dateTimeService.systemDateTime(ZoneId.of("UTC"))
-      systemDateTime must not be null
+      systemDateTime must not be None
```

Comment:
Please replace it with below as type of systemDateTime and None are not compatible. Same is applicable for 
line 35, 46 and 51

Option(systemDateTime) must not be empty